plugins/module_utils/dcim.py

```diff
@@ -54,6 +54,40 @@
 
 
 class NautobotDcimModule(NautobotModule):
+    def _find_software_version(self, version_identifier):
```

Comment:
None of this should be required. I would review #552 where I added `software_version` (among other fields) to the inventory item module and replicate what I did.